addons/web/controllers/main.py

```diff
@@ -588,6 +588,31 @@ def translations(self, mods=None, lang=None):
         return {"modules": translations_per_module,
                 "lang_parameters": lang_params}
 
+    @http.route('/web/webclient/technical_translations', type='json', auth="user")
+    def technical_translations(self, model):
+        fields = request.env['ir.model.fields'].search([('model', '=', model)])
+        trans = request.env['ir.translation'].search([
+            '|',
+                '&',('type', '=', 'model'),
+                    '&',('res_id', 'in', fields.ids),
+                        '|',
+                            ('name', '=', 'ir.model.fields,field_description'),
+                            ('name', '=', 'ir.model.fields,help'),
+                '&',('type', '=', 'selection'),
+                    ('name', 'ilike', model),
+        ])
+        view = request.env.ref("base.view_translation_tree", False)
+        return {
+            'name': _("Technical Translation"),
+            'view_mode': 'tree',
+            'views': [(view and view.id or False, "list")],
```

Comment:
For the return format of `views`, it should be `list`, not `tree`. cf [view_list.js](https://github.com/odoo/odoo/blob/365e298fb1907085d4071285c2be5dbcd5c59a0e/addons/web/static/src/js/views/list_view.js#L962), the tree is the old view like the chart of account in v7